eclass/eutils.eclass

```diff
@@ -15,14 +16,14 @@
 # different from the overall eclass!
 
 if [[ -z ${_EUTILS_ECLASS} ]]; then
-_EUTILS_ECLASS=1
```

Comment:
The `_EUTILS_ECLASS=1` must go before any inherit, in order to prevent inherit loops.

IIRC we had discussed this before, the general structure should look like this:
```
if [[ -z ${_FOO_ECLASS} ]]; then
_FOO_ECLASS=1
inherit bar baz quux
# main eclass content goes here
fi